content/contributors/cong-nguyen-dinh/index.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Cong NguyenßDinh 
```

Comment:
This rendered as the German ß, but your name on LinkedIn is written with a hyphen, "Cong Nguyen-Dinh". Did you mean to use a `-`, here?
```suggestion
name: Cong Nguyen-Dinh 
```